test/state/precompiles.cpp

```diff
@@ -272,7 +283,7 @@ inline constexpr auto traits = []() noexcept {
 #ifdef EVMONE_PRECOMPILES_SILKPRE
     // tbl[static_cast<size_t>(PrecompileId::ecrecover)].execute = silkpre_ecrecover_execute;
     tbl[static_cast<size_t>(PrecompileId::sha256)].execute = silkpre_sha256_execute;
-    tbl[static_cast<size_t>(PrecompileId::ripemd160)].execute = silkpre_ripemd160_execute;
+    // tbl[static_cast<size_t>(PrecompileId::ripemd160)].execute = silkpre_ripemd160_execute;
```

Comment:
why is this disabled?

Reply:
These override evmone's implementations of precompiles with silkpre. This does not make sense any more for RIPEMD160 by default as evmone just got the native implementation which is faster.

Reply:
maybe remove it completely instead of commenting?

Reply:
This section gives the visualization what has been replaced so far. The goal is get rid of silkpre dependency.